### ISSUE TITLE
Bug in CoordConv?

### ISSUE THREAD

**Issue by CuriousCat-7** [SeungyounShin/Adaptive-Wing-Loss-for-Robust-Face-Alignment-via-Heatmap-Regression#3]
https://github.com/SeungyounShin/Adaptive-Wing-Loss-for-Robust-Face-Alignment-via-Heatmap-Regression/blob/136e8749b546e3a0fa72916c9f828cc01162164e/models/CoordConv.py#L49
Kindly mention that:
```
ret = torch.cat([ret, xx_channel, yy_channel], dim=1)
```
should be 
```
ret = torch.cat([ret, xx_boundary_channel, yy_boundary_channel], dim=1)
```
?

**Comment by SeungyounShin** [comment]
you are right. I have big mistake. I will update it as soon as possible.
